app/upgrade.go

```diff
@@ -65,15 +65,19 @@ func (app *App) registerNagquUpgradeHandler() {
 	app.UpgradeKeeper.SetUpgradeInitializer(upgradetypes.Nagqu,
 		func() error {
 			app.Logger().Info("Init Nagqu upgrade")
-			mm, ok := app.mm.Modules[paymenttypes.ModuleName].(*paymentmodule.AppModule)
-			if !ok {
-				panic("*paymentmodule.AppModule not found")
-			}
-			mm.SetConsensusVersion(2)
+			app.setPaymentConsensusV2()
```

Comment:
prefer to `app.setPaymentConsensusVersion(2)` so that the function can be reused in the future.

Reply:
fixed